debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerChangeEventSourcePartitionMetricsMXBean.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.sqlserver;
+
+/**
+ * Metrics scoped to a source partition that are common for both snapshot and streaming change event sources
+ */
+public interface SqlServerChangeEventSourcePartitionMetricsMXBean extends SqlServerChangeEventSourceMetrics {
```

Comment:
This is not crystal clear at the moment.

The distribution of the meters between the partition and task metrics depends on the nature of the connector. For instance, "is connected?" belongs to the task metrics for SQL Server (one shared connection for all databases) but will likely belong to the partition metrics for MongoDB (one connection per replica set).

It will be clearer once this API is adopted by other connectors. Maybe we'll end up introducing higher-level blocks to build connector-specific interfaces from. E.g.:

```
SQL Server Task Metrics = Connected + Queue
SQL Server Partition Metrics = Monitored Tables

MongoDB Task Metrics = Queue
MongoDB Partition Metrics = Connected + Monitored Tables
```
Each meter class will implement the corresponding interface as well.